website/hugo.toml

```diff
@@ -167,16 +167,6 @@ enable = false
   url = "https://slack.cncf.io/"
   icon = "fab fa-slack"
   desc = "Chat with other project developers"
-[[params.links.user]]
-  name = "Discussion"
-  url = "https://github.com/lima-vm/lima/discussions"
-  icon = "fab fa-github"
-  desc = "Discussion and help from your fellow users"
-[[params.links.user]]
-  name = "Slack"
-  url = "https://slack.cncf.io/"
-  icon = "fab fa-slack"
-  desc = "Chat with other project users"
 
```

Comment:
https://deploy-preview-1842--lima-vm.netlify.app/community/

> ## Learn and Connect
> Using or want to use Lima? Find out more here:

↑ This sections is now empty

Reply:
yes it is empty now. I thought the idea was to remove multiple instances of the icons?